browser/about_flags.cc

```diff
@@ -340,10 +340,11 @@
 #endif  // BUILDFLAG(IS_ANDROID)
 
 #if !BUILDFLAG(IS_ANDROID)
-#define BRAVE_SHARED_PINNED_TABS                                        \
-  {"brave-shared-pinned-tabs", "Shared pinned tab",                     \
-   "Pinned tabs are shared across windows", kOsWin | kOsMac | kOsLinux, \
-   FEATURE_VALUE_TYPE(tabs::features::kBraveSharedPinnedTabs)},
+#define BRAVE_SHARED_PINNED_TABS                                            \
+  EXPAND_FEATURE_ENTRIES(                                                   \
+      {"brave-shared-pinned-tabs", "Shared pinned tab",                     \
+       "Pinned tabs are shared across windows", kOsWin | kOsMac | kOsLinux, \
+       FEATURE_VALUE_TYPE(tabs::features::kBraveSharedPinnedTabs)}),
```

Comment:
an extra comma will format this slightly better:
`kBraveSharedPinnedTabs)` -> `kBraveSharedPinnedTabs),`

Reply:
Oh, Nice!